firebase/database.rules.json

```diff
@@ -43,6 +43,13 @@
         ".write": false,
         ".read": true
       },
+      "teams": {
+        ".write": "auth.token.projectManager === true",
+        ".read": "auth.token.projectManager === true",
```

Comment:
@laurentS this might be a discussion point for the app side as well: do we plan to show the name of a team to the users of the app? currently only project managers can get the name from firebase.

Reply:
I think it would be good to show it, just as a verification that everything is working ok. I suspect some users might be anxious that they're correctly registered as part of the team, if their stats depend on it. We could just add a line in the "profile" screen saying `Your team: xxxxx`. It would probably save project managers a bunch of emails asking to confirm team memberships. Also, would project names include the team name in them? I suspect it might help if we can add tag somewhere on the project to say it is part of "team X".

Reply:
okay, sounds good. I can then adjust the rules accordingly.
For the projects we only add the `teamId` not the  `teamName`, but once you have access to the teams you could easily display it in the app as well at the project overview page. 
However, we don't want that users can access the `teamToken`, right?

Reply:
I can't think of a situation where a user who's already part of a team would need the token, since its purpose was for registration only. If you're able to restrict access to the token, great, but if it's too complicated with firebase rules, then I wouldn't worry too much about it. In theory, if a user is part of a team, it is because they already had the token in the first place :) (I'm oversimplifying, as that ignores token changes)

Reply:
with the new firebase db rules, users can only read the `teamName` attribute of the team they are a member of